scripts/install-test-dependencies.sh

```diff
@@ -1,6 +1,8 @@
 #!/bin/bash
 set -e
 
+pip3 uninstall -y fetchai-ledger-api
```

Comment:
Do we understand why `--force-resintall` is not covering this case. Honestly, I am not sure that I have run into this issue but maybe I am special! 😁